models/layerzero/layerzero_chain_list.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(
+    schema='layerzero',
+    alias = 'chain_list',
```

Comment:
We can add a static tag here.
```suggestion
    alias = 'chain_list',
    tags = ['static']
```

Reply:
Updated.